kimchi/src/circuits/expr.rs

```diff
@@ -468,20 +546,20 @@ impl FeatureFlag {
 /// the corresponding combination of the polynomials corresponding to
 /// the above variables should vanish on the PLONK domain.
 #[derive(Clone, Debug, PartialEq)]
-pub enum Expr<C> {
+pub enum Expr<C, Col: ColTrait = Column> {
```

Comment:
I am a bit confused aboout this notation. 
1. What exactly was `C`?
2. Is `Column` the kind of "columns" we have in Berkeley Kimchi, and if we had a different configuration with a different system we would have to re-write a new `Expr` whose `ColTrait` was `ColumnNew` for example?